index.bs

```diff
@@ -272,6 +272,9 @@ spec: HR-TIME; urlPrefix: https://w3c.github.io/hr-time/
 spec: RFC4648; urlPrefix: https://datatracker.ietf.org/doc/html/rfc4648
   type: dfn
     text: Base64 Encode; url: section-4
+spec: CSS-OVERFLOW-3; urlPrefix: https://drafts.csswg.org/css-overflow-3/
+  type: dfn
+    text: set scrollbar type; url: #set-scrollbar-type
```

Comment:
Did you maybe reference the wrong spec here? I cannot find the section for setting the scrollbar type and the anchor isn't there as well.

Reply:
That method I introduced in another pr, and it is blocked on this one

https://github.com/w3c/csswg-drafts/pull/13240